sdk/appconfiguration/app-configuration/samples/react-sample/src/components/Page.tsx

```diff
@@ -0,0 +1,160 @@
+/*
+  Copyright (c) Microsoft Corporation.
+  Licensed under the MIT license.
+*/
+
+import React from "react";
+import { useEffect, useState } from "react";
+import {
+  Navbar,
+  Nav,
+  NavDropdown,
+  Jumbotron,
+  Alert,
+  Row,
+  Col,
+  Container
+} from "react-bootstrap";
+import { getEnvironmentVariable } from "../utils";
+import {
+  AppConfigurationClient,
+  featureFlagPrefix,
+  isFeatureFlag,
+  isFeatureFlagClientFilter
+} from "@azure/app-configuration";
+
+let client: AppConfigurationClient;
+
+const featureFlag1 = {
+  id: "react-app-feature-1",
+  description: "",
+  enabled: false,
+  conditions: {
+    client_filters: []
+  }
+};
+const featureFlag2 = {
+  id: "react-app-feature-2",
+  description: "",
+  enabled: true,
+  conditions: {
+    client_filters: [
+      {
+        name: "Microsoft.TimeWindow",
+        parameters: {
+          Start: "Mon, 12 Apr 2021 07:00:00 GMT",
+          End: "Wed, 14 Apr 2021 07:00:00 GMT"
+        }
+      }
+    ]
+  }
+};
+
+export default function Page(): JSX.Element {
+  const feature1 = "react-app-feature-1";
+  const feature2 = "react-app-feature-2";
+  const [feature1ClassName, setFeature1ClassName] = useState<string>("feature-button-hidden");
+  const [feature2ClassName, setFeature2ClassName] = useState<string>("feature-button-hidden");
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const getFeatureFlags = async (keys: string[]): Promise<void> => {
+    const [setting1, setting2] = await Promise.all(
+      keys.map((key: string) =>
+        client.getConfigurationSetting({
+          key: featureFlagPrefix + key
+        })
+      )
+    );
+    if (isFeatureFlag(setting1)) {
+      console.log(`${setting1.key} is enabled : ${setting1.enabled}`, setting1);
+      setFeature1ClassName(setting1.enabled ? "feature-button-visible" : "feature-button-hidden");
+    }
+    if (isFeatureFlag(setting2)) {
+      console.log(`${setting2.key} is enabled : ${setting2.enabled}`, setting2);
+      const clientFilter = setting2.conditions.clientFilters?.[0];
+      if (isFeatureFlagClientFilter("timeWindow", clientFilter)) {
+        const now = Date.now();
+        const withinRange =
+          now - Date.parse(clientFilter.parameters.start) > 0 &&
+          Date.parse(clientFilter.parameters.end) - now > 0;
+        setFeature2ClassName(withinRange ? "feature-button-visible" : "feature-button-hidden");
+      }
+    }
+  };
+
+  useEffect(() => {
+    const connectionString = getEnvironmentVariable("REACT_APP_APPCONFIG_CONNECTION_STRING");
+    client = new AppConfigurationClient(connectionString);
+    getFeatureFlags([feature1, feature2]);
+  }, [getFeatureFlags]);
+
+  return (
+    <React.Fragment>
+      <Navbar bg="light" expand="xl">
```

Comment:
Personally I prefer just fetching the bootstrap assets from cdn and using classes with normal html elements instead of using `react-bootstrap`. It's going to be easier for folks who are not familiar with Bootstrap to understand our samples.

See here: https://getbootstrap.com/docs/5.0/getting-started/introduction/#css

Unless you need animation you can probably just pull the css, not the js.